thirdparty/build-thirdparty.sh

```diff
@@ -709,6 +709,7 @@ build_fmt() {
     mkdir -p build && cd build
     cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} ../
     make -j$PARALLEL && make install
+    cp -f ${TP_INSTALL_DIR}/lib/libfmt.a ${TP_INSTALL_DIR}/lib64/libfmt.a
```

Comment:
Need to check if ${TP_INSTALL_DIR}/lib/libfmt.a exists.
If libfmt.a was generated in lib64 in some platform, this cp operation will generate an error

Reply:
ok.I already added file check